src/vm/gdbjit.cpp

```diff
@@ -661,15 +661,57 @@ void __attribute__((noinline)) __jit_debug_register_code() { __asm__(""); };
 /* Make sure to specify the version statically, because the
    debugger may check the version before we can set it.  */
 struct jit_descriptor __jit_debug_descriptor = { 1, 0, 0, 0 };
+static CrstStatic JitDescriptorCrst;
```

Comment:
Coding convention for static variables in coreclr is `g_` prefix. Only static constants don't usually have this prefix. Could you please rename this instance to g_jitDescriptorCrst?

Reply:
done